docs/installation.rst

```diff
@@ -14,7 +14,13 @@ or using Poetry:
 Using with `starkware-libs/crypto-cpp`
 --------------------------------------
 
-To use the CPP library:
+By default, ``crypto-cpp`` is used and so it must be configured properly:
```

Comment:
This is only needed for local development, package in pypi has it included. info about `DISABLE_CRYPTO_C_EXTENSION` should added outside of installation.